pkg/config/generate.go

```diff
@@ -319,6 +319,14 @@ func Generate(cs *acsapi.ContainerService, c *Config) (err error) {
 		{
 			secret: &c.RegistryHTTPSecret,
 		},
+		{
+			secret: &c.RegistryConsoleOAuthSecret,
+			n:      68,
+		},
+		{
+			secret: &c.RouterStatsPassword,
```

Comment:
upstream sets [a-zA-Z0-9]{10}, then no base64 encode
https://github.com/openshift/origin/blob/master/pkg/oc/cli/admin/router/router.go#L1083-L1093

Reply:
ok, will revert as this was initial implementation.